packages/@aws-cdk/aws-codepipeline/test/integ.pipeline-code-deploy.expected.json

```diff
@@ -48,6 +48,14 @@
             "Arn"
           ]
         },
+        "AutoRollbackConfiguration": {
```

Comment:
Did you specify `false` to all three? Why are they all enabled?

Also, add a unit test

Reply:
> Did you specify false to all three? Why are they all enabled?

Those are actually 2 different tests :) I've specified `false` for all 3 in the `integ.deployment-group`, while this is `integ.pipeline-code-deploy`.

> Also, add a unit test

Done.